Formula/spice-gtk.rb

```diff
@@ -0,0 +1,86 @@
+class SpiceGtk < Formula
+  desc "GTK client/libraries for SPICE"
+  homepage "https://www.spice-space.org"
+  url "https://www.spice-space.org/download/gtk/spice-gtk-0.36.tar.bz2"
+  sha256 "7126c3584df12d40dcf4970f5f46fafd65b757620f911687e549f7da5c8fd7cd"
+
+  depends_on "autoconf" => :build
+  depends_on "autogen" => :build
+  depends_on "automake" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "intltool" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vala" => :build
+
+  depends_on "atk"
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gst-libav"
+  depends_on "gst-plugins-bad"
+  depends_on "gst-plugins-base"
+  depends_on "gst-plugins-good"
+  depends_on "gst-plugins-ugly"
+  depends_on "gstreamer"
+  depends_on "gtk+3"
+  depends_on "jpeg"
+  depends_on "json-glib"
+  depends_on "libusb"
+  depends_on "lz4"
+  depends_on "openssl"
+  depends_on "opus"
+  depends_on "pango"
+  depends_on "pixman"
+  depends_on "spice-protocol"
+  depends_on "usbredir"
+
+  def install
+    # Some files (vncdisplaykeymap.c) require building as Objective-C
+    # https://www.mail-archive.com/spice-devel@lists.freedesktop.org/msg40085.html
+    ENV["CFLAGS"] = "-ObjC -g -O2"
```

Comment:
- `-g -O2` are handled by Homebrew, do not add those.
- `-ObjC` should be given by the configure/Makefile process, really. Has this bug been reported?

Reply:
Reported [upstream](https://gitlab.freedesktop.org/spice/spice-gtk/issues/88) with proposed patch. Does patch need to be accepted upstream before applying to formula?

Reply:
Upstream reported/pushed issue to a dependency, gtk. Issue [here](https://gitlab.gnome.org/GNOME/gtk/issues/1737). Patch and merge request [here](https://gitlab.gnome.org/GNOME/gtk/merge_requests/641).

Once gtk+3 is updated, then this formula can be updated.